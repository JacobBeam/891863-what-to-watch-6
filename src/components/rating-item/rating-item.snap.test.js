import React from 'react';
import {render} from '@testing-library/react';

import RatingItem from './rating-item';

const id =1
const onChangeInput = jest.fn();
const isSentComment = true;
it(`Should RatingItem render correctly`, () => {
  const { container } = render(
    <RatingItem
      id={id}
      onChangeInput={onChangeInput}
      isSentComment={isSentComment} />
  );
  expect(container).toMatchSnapshot();
});
