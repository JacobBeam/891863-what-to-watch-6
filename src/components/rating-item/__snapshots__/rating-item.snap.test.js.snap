// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should RatingItem render correctly 1`] = `
<div>
  <input
    class="rating__input"
    disabled=""
    id="star-1"
    name="rating"
    type="radio"
    value="1"
  />
  <label
    class="rating__label"
    for="star-1"
  >
    Rating 
    1
  </label>
</div>
`;
